crates/zed/src/languages/go.rs

```diff
@@ -46,11 +46,11 @@ impl super::LspAdapter for GoLspAdapter {
         delegate: &dyn LspAdapterDelegate,
     ) -> Result<Box<dyn 'static + Send + Any>> {
         let release = latest_github_release("golang/tools", false, delegate.http_client()).await?;
-        let version: Option<String> = release.name.strip_prefix("gopls/v").map(str::to_string);
+        let version: Option<String> = release.tag_name.strip_prefix("gopls/v").map(str::to_string);
```

Comment:
https://github.com/golang/tools/releases/tag/gopls%2Fv0.14.2

Here is ok, `tag_name` in `golang/tools` is `gopls/v0.14.2`